Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitStorageTypeOptionsForCluster.java

```diff
@@ -75,6 +75,11 @@
             SparkSubmitStorageType.DEFAULT_STORAGE_ACCOUNT
     }),
 
+    // arcadia cluster now suppports blob
+    ArcadiaSparkCluster(new SparkSubmitStorageType[]{
+            SparkSubmitStorageType.BLOB
+    }),
```

Comment:
Maybe we need to specify `getDefaultStorageType` and  `getStorageOptionsType` for class `ArcadiaSparkCompute`.